setup_micro.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+import setuptools
+
+loc = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_description():
+    with open(loc + '/docs/README.rst') as readme:
+        info = readme.read()
+    with open(loc + '/docs/changelog.rst') as changelog:
+        return info + '\n\n' + changelog.read()
+
+
+with open(loc + '/requirements-micro.txt') as f:
+    required = f.read().splitlines()
+
+with open('flask_monitoringdashboard/constants.json', 'r') as f:
+    constants = json.load(f)
+
+setuptools.setup(
+    name="Flask-MonitoringDashboard-Micro",
```

Comment:
Is it necessary to add a new project name in PyPI?

Reply:
Beyond having a new project name, there's a lot of duplication between files now.
setup_micro.py and requirements_micro have to always be changed with the corresponding files, and that is not cool. Can you think of a better solution?

Reply:
RE: Adding new pypi project - Probaly? I have my own that I use and it needed a seperate wheel for sure. I have no idea if there is a way to do versions of your project. I think i saw someone do weird stuff with []s before though so could be a possible avenue to persue.

For the other point you are right. We could in theory import setup.py into setup micro and then modify it appropriatly in setup_micro - I assume we can pass an exploded dict to .setup. Alternatively we could have a common file that gets imported and modified by both. 

Requirements wise - probably the same deal - have a common reqs file and then a micro and standard might work?

I kinda took the path of least resistance for this since I wanted to use it myself right away and assumed you would have some feedback.